src/pages/workspace/WorkspaceInviteMessagePage.tsx

```diff
@@ -186,6 +206,16 @@ function WorkspaceInviteMessagePage({policy, route, currentUserPersonalDetails}:
                         <Text>{translate('workspace.inviteMessage.inviteMessagePrompt')}</Text>
                     </View>
                     <View style={[styles.mb3]}>
+                        <View style={[styles.mhn5, styles.mb6]}>
```

Comment:
So I think this actually needs to be mb3... 24px is the visual gap we want below the bottom part of the text in the role row... but that row already has 12px vertical padding. So if we change this to mb3 (12px), we get this, which is desired:
![CleanShot 2025-01-14 at 15 58 45@2x](https://github.com/user-attachments/assets/801da5f1-2a80-41d2-827d-111bab0393b0)

Reply:
Looks good, thanks!